src/styles/Replay.css

```diff
@@ -0,0 +1,50 @@
+.Replay {
+  display: flex;
+  flex-direction: column;
+  min-height: 700px;
+  height: 100vh;
+}
+
+.Replay-Content {
+  /* margin: 100px 0px 0px 0px; */
```

Comment:
any reason to keep this in as a comment?

Reply:
whoops, I thought I removed it. Fixed now